uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/Service/WorkPatternCalendar.php

```diff
@@ -154,6 +129,8 @@ private function getWorkPatternsPeriods() {
       }
     }
 
+    $workPatterns = $this->fillWorkPatternPeriodsLapses($workPatterns);
```

Comment:
since now this method behaves a little bit differently, you need to update the docblock to reflect that. Please also to the same to all the other methods you have changed